public/components/loginForm.jsx

```diff
@@ -0,0 +1,24 @@
+import css from '../css/login.module.css';
+export default function LoginForm() {
+    return (
+        <main className={css.mainWrapper}>
```

Comment:
main should be ONLY in main (index). If you want to have a wrapper to the component/page add it as a div with a respective class/id.